packages/admin-next/dashboard/src/routes/product-variants/product-variant-detail/components/variant-general-section/variant-general-section.tsx

```diff
@@ -0,0 +1,97 @@
+import { ProductVariantDTO } from "@medusajs/types"
+import { Badge, Container, Heading, usePrompt } from "@medusajs/ui"
+import { Component, PencilSquare, Trash } from "@medusajs/icons"
+import { useTranslation } from "react-i18next"
+import { useNavigate } from "react-router-dom"
+
+import { ActionMenu } from "../../../../../components/common/action-menu"
+import { SectionRow } from "../../../../../components/common/section"
+import { useDeleteVariant } from "../../../../../hooks/api/products"
+
+type VariantGeneralSectionProps = {
+  variant: ProductVariantDTO
+}
+
+export function VariantGeneralSection({ variant }: VariantGeneralSectionProps) {
+  const { t } = useTranslation()
+  const prompt = usePrompt()
+  const navigate = useNavigate()
+
+  const hasInventoryKit = variant.inventory?.length > 1
+
+  const { mutateAsync } = useDeleteVariant(variant.product_id, variant.id)
+
+  const handleDelete = async () => {
+    const res = await prompt({
+      title: t("general.areYouSure"),
+      description: t("products.variant.deleteWarning", {
+        title: variant.title,
+      }),
+      confirmText: t("actions.delete"),
+      cancelText: t("actions.cancel"),
+    })
+
+    if (!res) {
+      return
+    }
+
+    await mutateAsync(undefined, {
+      onSuccess: () => {
+        navigate("..")
```

Comment:
```suggestion
        navigate("..", { replace: true })
```
to prevent the user from going back using the browser navigation